ios/Podfile.lock

```diff
@@ -287,6 +289,7 @@ DEPENDENCIES:
   - React/RCTWebSocket (from `../node_modules/react-native`)
   - ReactNativeFabric (from `../node_modules/react-native-fabric`)
   - ReactNativePermissions (from `../node_modules/react-native-permissions`)
+  - "RNCAsyncStorage (from `../node_modules/@react-native-community/async-storage`)"
```

Comment:
```suggestion
  - RNCAsyncStorage (from `../node_modules/@react-native-community/async-storage`)
```

Reply:
P. S. this change might cause different checksum of file, so might need to update it

Reply:
I just ran `pod install` locally - lock file looks unchanged

Reply:
After I changed to `RNCAsyncStorage (from `../node_modules/@react-native-community/async-storage`)` and re-ran `pod install` it returned me back the quotes. Maybe it's because of `@` in the package name? Or maybe it's because of my new xcode verison..

Reply:
could be due XC, nevermind, this change is not crucial, just look cleanier